crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -25,6 +25,8 @@ use bevy_reflect::Reflect;
 /// This system runs in stage [`CoreStage::PostUpdate`](crate::CoreStage::PostUpdate). If you
 /// update the [`Transform`] of an entity in this stage or after, you will notice a 1 frame lag
 /// before the [`GlobalTransform`] is updated.
+/// 
+/// Example: [transform](https://github.com/bevyengine/bevy/blob/latest/examples/transforms/transform.rs),
```

Comment:
I think the global vs local transform example is important to call out here

Reply:
whoops! meant to put that there, changed it.